src/main/java/com/meilisearch/sdk/exceptions/APIError.java

```diff
@@ -21,4 +21,22 @@ public APIError(String message, String code, String type, String link) {
         this.type = type;
         this.link = link;
     }
+
+    @Override
+    public String toString() {
+        return "APIError: {"
```

Comment:
I have some different feelings when I see this haha, but I don't know to improve it :|

Reply:
Me neither it was already like this ...